src/PythonBindings/VectorPyList.hpp

```diff
@@ -11,13 +11,28 @@
 
 namespace bp = boost::python;
 
+/*!
+ * \ingroup PythonBindingsGroup
+ *\brief A namespace containing functions for binding Python wrappers
+ *to their respective c++ classes and functions, as well as helper functions for
+ *these wrappers.
```

Comment:
Please add a space between `*` and the text

Reply:
@nilsleiffischer fixed and squashed